src/System.IO.MemoryMappedFiles/tests/Performance/Perf.MemoryMappedFile.cs

```diff
@@ -18,6 +18,7 @@ public class Perf_MemoryMappedFile : MemoryMappedFilesTestBase
         [InlineData(100000)]
         [InlineData(1000000)]
         [InlineData(10000000)]
+        [ActiveIssue(18463, TestPlatforms.OSX)] // Hits the max-open-file limit often on OSX.
```

Comment:
Fine for now, but I don't know that the separation of the creation from the disposal is particular interesting, and it's that separation that's necessitating many maps be open at the same time.